src/tests/unit/tests/popup/scripts/actions/popup-action-message-creator.test.ts

```diff
@@ -222,4 +219,24 @@ describe('PopupActionMessageCreatorTest', () => {
         postMessageMock.verifyAll();
         telemetryFactoryMock.verifyAll();
     });
+
+    test('setLaunchPanelType', () => {
+        const type = LaunchPanelType.AdhocToolsPanel;
+
+        const payload: SetLaunchPanelState = {
+            launchPanelType: type,
+        };
+
+        const expectedMessage = {
+            tabId: 1,
+            type: Messages.LaunchPanel.Set,
+            payload,
+        };
+
+        postMessageMock.setup(post => post(It.isValue(expectedMessage))).verifiable(Times.once());
+
+        testSubject.setLaunchPanelType(type);
+
+        postMessageMock.verifyAll();
```

Comment:
Its not a blocker or anything, but we can move the `verifyAll` calls to `afterEach()`. That just keeps us from calling this function in each test!

Reply:
I think `afterEach` should be use for clean up purposes only... `verifyAll` is part of the last step of AAA testing so I prefer it to be explicitly part of the test itself.